### ISSUE TITLE
rocket.chat

### ISSUE THREAD

**Issue by TinajaLabs** [guidone/node-red-contrib-chatbot#76]
I use rocket.chat (very much like slack) and hubot.

Is there a way to use this with Rocket.chat?  And if so, would you please suggest where I might hook into this lib to make it work.

Thanks,
Chris.

**Comment by guidone** [comment]
Hi @TinajaLabs, no I don't support rocket chat yet. Sorry for the delay of the answer.

You might give a look to `chatbot-telegram-receive.js`, if there's a nodejs library for rocket chat then you can duplicate this node, rename the ids here and there, and start from that.

Basically it's a big switch code, it transform what the specific platform libraries return into a uniformed payload. The same is done when one of these payload goes out through a specific platform.
This makes the flow reusable for different platforms.
Basically a message "hello word" from Facebook will have - in RedBot - the same payload of a "hello world" message coming from Telegram, except for the `transport` property.
